src/actix/api/service_api.rs

```diff
@@ -23,15 +26,23 @@ use crate::common::telemetry::TelemetryCollector;
 use crate::settings::ServiceConfig;
 use crate::tracing;
 
-#[derive(Deserialize, Serialize, JsonSchema)]
+#[derive(Deserialize, Serialize, JsonSchema, Validate)]
 pub struct TelemetryParam {
     pub anonymize: Option<bool>,
     pub details_level: Option<usize>,
+    #[validate(range(min = 1, max = 10))]
```

Comment:
Let's set the maximum at something more unreasonable, like 1 hour.

I've seen valid cases where fetching telemetry took 12 seconds. It was on a (very!) busy cluster with thousands of collections, shards and segments.